pkg/ddc/jindo/label.go

```diff
@@ -5,3 +5,7 @@ import "github.com/fluid-cloudnative/fluid/pkg/common"
 func (e *JindoEngine) getCommonLabelname() string {
 	return common.LabelAnnotationStorageCapacityPrefix + e.namespace + "-" + e.name
 }
+
+func (e *JindoEngine) getFuseLabelname() string {
```

Comment:
Do we need to add this label to Jindo helm chart?

Reply:
The fuse label will be transformed into the fuse daemonSet. So no need to add this label explicitly to Jindo helm chart.